_docs/_latest/concepts/models.md

```diff
@@ -1,84 +1,80 @@
 ---
 title: Models
-order: 100
+order: 003
 ---
 
 # {{ page.title }}
 
 Starting with version 2.0, Forseti introduces the use of data models.
 
 The data model is an additional pool of data that is relational in nature,
-and is created from the flat json data in inventory, that's captured from GCP
-API responses. With the relational data, we can more easily see the
-relationships and the inheritances between resources, as well as being able
-to query for them.  One example is to see all the accesses that a service
-account may have, across all the projects.
+and is created from the flat json data in inventory. With the relational data, Forseti 
+can more easily understand the entire relationship, including inheritance between resources. Models
+allow for easier querying against the entire computed policy. For example, a data model for
```

Comment:
"a data model for service accounts" sounds a bit weird, I think the relationships for all the resources are stored in one big data model instead of having one per resource.

Reply:
Agreed, removed for now.